# An array of ClassifiedObject messages

ClassifiedObject[] objects

#The classified image
sensor_msgs/Image image
#The CameraInfo of the camera used to capture the image.
sensor_msgs/CameraInfo camera_info